gallery/src/components/NxTabs/NxTabsSimpleExample.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel
+} from '@sonatype/react-shared-components';
+
+const NxTableSimpleExample = () => {
+  const [activeTabId, setActiveTabId] = useState('tab-1');
+
+  function handleKeyPress(event: React.KeyboardEvent<HTMLLIElement>) {
+    if (event.key === ' ') {
+      event.preventDefault();
+      setActiveTabId(event.currentTarget.id);
+    }
+  }
+
+  function handleClick(event: React.MouseEvent<HTMLLIElement, MouseEvent>) {
+    setActiveTabId(event.currentTarget.id);
+  }
+
+  return (
+    <NxTabs activeTab={activeTabId}>
+      <NxTabList aria-label="Simple Tabs">
+        <NxTab id="tab-1" tabIndex={1} onKeyPress={handleKeyPress} onClick={handleClick}>Tab 1</NxTab>
```

Comment:
I'm not a big fan of the tab labels being DOM ids rather than some react-only prop that we make up, considering the uniqueness requirements on ids.